registrations/tasks.py

```diff
@@ -1,23 +1,225 @@
+import datetime
+import six  # for Python 2 and 3 string type compatibility
+
 from celery.task import Task
 from celery.utils.log import get_task_logger
 
+from .models import Registration
+
 
 logger = get_task_logger(__name__)
 
 
-class The_Incr(Task):
+def get_today():
+    return datetime.today()
+
+
+def is_valid_date(date):
+    try:
+        datetime.datetime.strptime(date, "%Y%m%d")
+        return True
+    except:
+        return False
+
+
+def is_valid_uuid(id):
+    return len(id) == 36 and id[14] == '4' and id[19] in ['a', 'b', '8', '9']
+
+
+def is_valid_lang(lang):
+    return lang in ["english", "runyakore", "lusoga"]
+
+
+def is_valid_msg_type(msg_type):
+    return msg_type in ["sms"]  # currently sms-only
+
+
+def is_valid_msg_receiver(msg_receiver):
+    return msg_receiver in ["head_of_household", "mother_to_be",
+                            "family_member", "trusted_friend"]
+
+
+def is_valid_loss_reason(loss_reason):
+    return loss_reason in ['miscarriage', 'stillborn', 'baby_died']
 
+
+def is_valid_name(name):
+    return isinstance(name, six.string_types)  # TODO reject non-letters
```

Comment:
Can you open a new issues for TODO's? Maybe one for validations, one for calculations?